src/Laravel/Cashier/WebhookController.php

```diff
@@ -50,6 +50,16 @@ protected function eventExistsOnStripe($id)
     }
 
     /**
+     * Check if testing mode is on
+     * 
+     * @return bool
+     */
+    protected function isInTestingMode()
+    {
+        return (bool) getenv('CASHIER_TESTING');
```

Comment:
Maybe we should check if APP_ENV is equal to testing instead of adding yet another variable.